bundles/org.openhab.core/src/main/java/org/eclipse/smarthome/core/items/Metadata.java

```diff
@@ -35,6 +35,9 @@
     private final String value;
     private final Map<String, Object> configuration;
 
+    Metadata() { // For (de)serialization
+    }
```

Comment:
You need to assign a "dummy" value to the final fields to pass the null checks (all members are non null).
You will see an example in one of the referenced PR.

Reply:
True. Regarding this other branch, I'm using the github editor to get this done as another core checkout and build would poison my local maven cache.

Reply:
fixed. But you are rather verbose here in terms of LOC for this actually very obvious fact. I think I liked my one liner more.

Reply:
It has not been chosen by me (IIRC) but present in some ESH file for years.
My requirement is that it is the same for all files.

So, if we would like to search all files and replace it, I am still happy. :wink:

Reply:
> as another core checkout and build would poison my local maven cache.

Just if you are interested in: `mvn -Dmaven.repo.local=/some/other/directory ...`

Reply:
I'm a gradle user and so to say "forced" to use maven for this project. I turns out it is not 10 times slower as I originally thought (that was tycho) and if you turn it silent, it also doesn't cause so much noise anymore. But I'm still not familiar with all those command line switches. Thanks for the hint.